context/app/static/js/components/detailPage/provenance/ProvGraph/ProvGraph.jsx

```diff
@@ -23,69 +70,33 @@ function ProvGraph({ provData, entity_type, uuid }) {
     setUUID(uuid);
   }, [setUUID, uuid]);
 
-  function getNameForActivity(id, prov) {
-    const activity = prov.activity[id];
-    return `${activity['hubmap:creation_action']} - ${activity[idKey]}`;
-  }
-
-  function getNameForEntity(id, prov) {
-    const entity = prov.entity[id];
-    // NOTE: The initial entity node was not included in the sample data;
-    // Fallback to ID, if needed. https://github.com/hubmapconsortium/prov-vis/issues/15
-    return entity ? `${entity[typeKey]} - ${entity[idKey]}` : id;
-  }
+  const getNameForActivity = useCallback(
+    function getNameForActivity(id, prov) {
+      const activity = prov.activity[id];
+      return `${activity['hubmap:creation_action']} - ${activity[idKey]}`;
+    },
+    [idKey],
+  );
```

Comment:
Were the additions of callback here prompted by eslint?

Reply:
Removed the useCallbacks here, I think I added those before we agreed on our current heuristic.